clientconn.go

```diff
@@ -891,10 +891,18 @@ func (cc *ClientConn) healthCheckConfig() *healthCheckConfig {
 	return cc.sc.healthCheckConfig
 }
 
-func (cc *ClientConn) getTransport(ctx context.Context, failfast bool, method string) (transport.ClientTransport, func(balancer.DoneInfo), error) {
-	t, done, err := cc.blockingpicker.pick(ctx, failfast, balancer.PickInfo{
-		Ctx:            ctx,
-		FullMethodName: method,
+type getTransportOpts struct {
```

Comment:
Would it make sense to use (part of) `PickInfo` instead?

Reply:
It makes the call to pick a bit arkward, to send the missing fields of pickInfo:
```go
opts.pickInfo.Ctx = ctx
opts.pickInfo.Method = method
picker.Pick(opts.pickInfo)
```